packages/webview_flutter/webview_flutter_platform_interface/lib/src/types/over_scroll_mode.dart

```diff
@@ -0,0 +1,16 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// Defines the over-scroll behavior of a WebView.
+enum WebViewOverScrollMode {
+  /// Always allow a user to over-scroll the WebView.
+  always,
+
+  /// Allow a user to over-scroll the WebView only if the content is large
+  /// enough to meaningfully scroll.
```

Comment:
What does "meaningfully" mean here? Isn't the behavior of this mode just that it will allow overscroll any time the content is larger than the viewport?

Reply:
That is how I interpreted it as well. This language is similar to how Android describes it: https://developer.android.com/reference/android/view/View#OVER_SCROLL_IF_CONTENT_SCROLLS.

If it seems clearer to make this more technical by using "content is larger than the viewport" then I'm fine with switching it. I assumed most developers would interpret "meaningfully" the same way as we did.

Reply:
I would prefer we be explicit; I was confused by the current language (and the Android language) because I'm not sure what "meaningfully" is supposed to indicate here. I.e., what is the case where the content can scroll but not "meaningfully scroll"? I guess it means overscrolling?

Reply:
I think the "meaningfully scroll" only refers to the size of the content relative to the viewport. I think overscroll typically adds to the page to make it seem scrollable.